packages/api-demo/package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "containerjs-api-bundle": "0.0.2",
     "containerjs-api-electron": "0.0.2",
+    "containerjs-compatibility-api": "0.0.1",
```

Comment:
rather than adding to the existing api-demo, I'd prefer to keep this code self-container within `containerjs-compatibility-api` as I don't think we see these APIs being included in the long-term. Having it all in one place makes it easier to remove ;-)